docs/conf.py

```diff
@@ -16,6 +16,14 @@
 import sys
 import os
 
+# Define the branch reference for linkcode_resolve
+DOCS_BUILD_REF = os.environ.get("DOCS_BUILD_REF", "stable")
```

Comment:
🤔 LGTM! Around lines `19`-`20` the new variable `DOCS_BUILD_REF` is defined and initialized with an optional environment override. The 'stable' branch remains the default.